hack/godep-save.sh

```diff
@@ -78,7 +78,9 @@ GOPATH="${GOPATH}:$(pwd)/staging" ${KUBE_GODEP:?} save "${REQUIRED_BINS[@]}"
 
 # create a symlink in vendor directory pointing to the staging client. This
 # let other packages use the staging client as if it were vendored.
-for repo in $(ls staging/src/k8s.io); do
+for full_repo_path in staging/src/k8s.io/*; do
+  repo=$(basename ${full_repo_path})
```

Comment:
You're hitting another shellcheck failure:
```
In ./hack/godep-save.sh line 82:
  repo=$(basename ${full_repo_path})
                  ^---------------^ SC2086: Double quote to prevent globbing and word splitting.
```

More to the point, what was the original failure you were addressing, and why do it this way?

Reply:
Great question! I was fixing https://github.com/koalaman/shellcheck/wiki/SC2045, which says to prefer shell globs over iterating over `ls`. I'll also fixed the additional shellcheck failure - thanks for catching that :)